nginx/public/node/frontend/public/js/app/manuscript-detail/folio/chant-record.template.html

```diff
@@ -31,6 +31,9 @@
 <% if (differentia) { %>
 <p><b>Differentia: </b> <%= differentia.replace('w', 'ω') %></p>
 <% } %>
+<% if (differentiae_database) { %>
+    <p><b>Differentiae Database: </b> <%= differentiae_database %></p>
+<% } %>
```

Comment:
Not required before merging, but - I appreciate the indentation in these lines! It would really help readability if such indentation was applied to the rest of the file.

(also - what templating language is this? I'm confused by the javascripty parens-with-braces for conditionals, but wrapped in `<% %>`...)

Reply:
The template language is underscore's I believe (underscore is a dependency of marionette) `<% ... %>` executes the javascript code at `...`

https://underscorejs.org/#template